website/content/docs/release-notes/consul/v1_15_x.mdx

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+page_title: 1.15.x
+description: >-
+  Consul release notes for version 1.15.x
+---
+
+# Consul 1.15.0
+
+## Release Highlights
+
+- **Enhanced Envoy Access Logging:** Envoy access logs are now centrally managed via config entries and CRDs to allow operators to easily turn on access logs for all proxies within the service mesh. 
+Refer to [Proxy default configuration entry](https://developer.hashicorp.com/consul/docs/connect/config-entries/proxy-defaults) for more details on how to enable access logs centrally via the ProxyDefaults config entry or CRD.  
+
+- **Consul Envoy Extensions:** A new extension system has been added to Consul to modify Consul-generated Envoy resources outside of the Consul binary. This will allow extensions to add, delete, and modify Envoy listeners, routes, clusters, and endpoints, enabling support for additional Envoy features without changes to the Consul codebase.
```

Comment:
```suggestion
- **Consul Envoy Extensions:** The new Envoy extension system enables you to modify Consul-generated Envoy resources outside of the Consul binary. This will allow extensions to add, delete, and modify Envoy listeners, routes, clusters, and endpoints, enabling support for additional Envoy features without changes to the Consul codebase. 
```